crates/forge_tracker/src/collect/mod.rs

```diff
@@ -7,4 +7,9 @@ pub mod posthog;
 #[async_trait::async_trait]
 pub trait Collect: Send + Sync {
     async fn collect(&self, event: Event) -> super::Result<()>;
+    async fn identify(
+        &self,
+        distinct_id: String,
+        properties: std::collections::HashMap<String, serde_json::Value>,
```

Comment:
```suggestion
        user: User,serde_json::Value>,
```

Identify should take a new type called `User`. `User` should have all the necessary details to "identify" the profile.